auth_service/user_management/models/dto.py

```diff
@@ -98,9 +114,33 @@ class Config:  # pylint: disable=missing-class-docstring
         use_enum_values = True
 
 
+class UserAutomaticData(BaseModel):
+    """User data that is automatically created except the ID"""
+
+    registration_date: datetime = Field(default=..., title="Registration date")
+
+    status_change: StatusChange
```

Comment:
Maybe make this optional so that the fields in the StatusChange don't have to be Optional.

Reply:
This is now optional and only created when there is a real change of the status after the initial creation of the user.